index.html

```diff
@@ -1171,7 +1171,36 @@ <h3>Malicious Issuers and Verifiers</h3>
     </section>
 
     <section class="informative">
-      <h3>Multistatus Correlation</h3>
+      <h3>Status List Monitoring</h3>
+
+      <p>
+Once a <a>verifier</a> knows of a status list and entry index that is
+associated with a specific <a>holder</a>, it becomes possible for that
```

Comment:
I'm rejecting the `issuee` language until we have consensus to add the term in the VC Data Model. If we have that, then all of the specs will have to be updated to use that base terminology.

Reply:
I think this might be more accurate, and pass muster.
```suggestion
associated with a specific <a>holder</a> or <a>subject</a> , it becomes possible for that
```

Reply:
Yes, Thankyou.

Reply:
I think [*this* small change](https://github.com/w3c/vc-bitstring-status-list/pull/156/files#r1548247936) needs a new PR to get into the document. I *think* the other change suggestions have all been applied.

Reply:
@msporny can you do this please?

Reply:
See new PR #163